client/src/components/Sidebar/modules/Search.scss

```diff
@@ -53,15 +55,25 @@
         transition: opacity $menu-transition-duration ease, width $menu-transition-duration ease;
 
         svg {
-            margin-right: 15px;
-            transition: margin-right $menu-transition-duration ease;
+            margin-right: var(--svg-margin);
+            transition: margin-right $menu-transition-duration ease, margin-left $menu-transition-duration ease;
         }
 
         .sidebar--slim & {
             width: 100%;
 
             svg {
-                margin-right: 0;
+                --svg-margin: 0;
```

Comment:
Question : this is set initially on `&__submit`, couldn't it be overriden in `.sidebar--slim &` directly ?